keel-retrofit-test-support/src/main/kotlin/com/netflix/spinnaker/keel/retrofit/model/ModelParsingTestSupport.kt

```diff
@@ -28,7 +23,7 @@ import strikt.assertions.isNotNull
 import java.net.URL
 import java.nio.charset.Charset
 
-abstract class BaseModelParsingTest<out T> {
+abstract class ModelParsingTestSupport<in S, out E>(serviceType: Class<S>) {
```

Comment:
I’m surprised you even need the `in`

Reply:
the `in` was needed for the `call` signature of `S.() ->`